src/OmniSharp.MSBuild/Resolution/PackageDependencyResolver.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using Microsoft.Extensions.Logging;
+using NuGet.ProjectModel;
+using OmniSharp.MSBuild.ProjectFile;
+
+namespace OmniSharp.MSBuild.Resolution
+{
+    internal class PackageDependencyResolver
```

Comment:
I think there's definitely value in having tests, but I don't want to gate this PR on them. I want to write tests for the MSBuildProjectSystem in general because there's really not much today. Now that we have the test infrastructure in place, I'd like to write some broader MSBuildProjectSystem tests.